### PR TITLE
Update default value system and group names.

### PR DESCRIPTION
**Pull request by pshriwise**
This PR makes the default values found in the CLI client part of the `Gopt` struct part of it's default state upon creation. It also changes the default value for the `uwuw_names` option from `false` to `true` -- there aren't any workflows I'm aware of that support the old naming scheme any more.

### CODE REVIEW

File: mcnp2cad.cpp [hunk 1]
```diff
@@ -88,8 +88,7 @@ class GeometryContext {
   std::string materialName( int mat, double rho ){
     std::string ret;
     std::stringstream formatter;
-    if(Gopt.uwuw_names){
-      bool mass_density = false;
+       bool mass_density = false;
```
Comment (gonuke) [inline mcnp2cad.cpp:91]: The indentation seems inconsistent.

Also, I think the closing brace from this now-deleted if statement is still floating around somewhere?

**Review by gonuke — CHANGES_REQUESTED** [review]
It looks like there was one substantive change buried in all the whitespace noise, and I think it's missing a matching deletion of the closing brace.

File: mcnp2cad.cpp (hunk 1 above)
Comment (pshriwise) [reply mcnp2cad.cpp:91]: Ah, yeah that deletion was a mistake. Thanks! Just replaced that line and fixed the indentation.

**Comment by pshriwise** [comment]
> It looks like there was one substantive change buried in all the whitespace noise, and I think it's missing a matching deletion of the closing brace.

Thanks for the rapid review! Just took care of that mistakenly removed like. And apologies for the whitespace noise, my editor is set to aggressively remove trailing whitespace.

**Comment by gonuke** [comment]
BTW - who actually uses this anymore?  Was this prompted by user interest?

**Review by gonuke — APPROVED** [review]

**Comment by pshriwise** [comment]
Do you mean the CLI version?

**Pull request merged**